From 090d26723f7cdbb3aebfe212170b7f81c163dd7b Mon Sep 17 00:00:00 2001
From: Jesse Barnes <jbarnes@virtuousgeek.org>
Date: Thu, 8 Jul 2010 09:08:51 -0700
Subject: [PATCH 02/11] dixstruct: make ignoreCount change more ABI friendly

Move ignoreCount to the bottom of the ClientRec where it should have
less impact on the ABI.

Signed-off-by: Jesse Barnes <jbarnes@virtuousgeek.org>
---
 include/dixstruct.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git xorg-server-1.8.2/include/dixstruct.h xorg-server-1.8.2/include/dixstruct.h
index 568ea1f..5d24f0d 100644
--- xorg-server-1.8.2/include/dixstruct.h
+++ xorg-server-1.8.2/include/dixstruct.h
@@ -99,7 +99,6 @@ typedef struct _Client {
     int         clientGone;
     int         noClientException;	/* this client died or needs to be
 					 * killed */
-    int         ignoreCount;		/* count for Attend/IgnoreClient */
     SaveSetElt	*saveSet;
     int         numSaved;
     void	*unused_screenPrivate[16];
@@ -125,6 +124,7 @@ typedef struct _Client {
     long    smart_check_tick;
     
     DeviceIntPtr clientPtr;
+    int         ignoreCount;		/* count for Attend/IgnoreClient */
 }           ClientRec;
 
 /*
-- 
1.7.2.1

